torchao/quantization/quant_api.py

```diff
@@ -738,6 +746,25 @@ def _int8_symm_per_token_reduced_range_quant(x: torch.Tensor) -> torch.Tensor:
     )
 
 
+def _int8_symm_per_token_reduced_range_quant_cutlass(x: torch.Tensor) -> torch.Tensor:
```

Comment:
probably need to add this to https://github.com/pytorch/ao/blob/abff563ba515576fc48cd4ac0feb923dd65dc267/torchao/quantization/quant_api.py#L1170 for serialization support, can you add cutlass int4 layout to https://github.com/pytorch/ao/blob/abff563ba515576fc48cd4ac0feb923dd65dc267/test/dtypes/test_affine_quantized.py#L34 so it's tested for serialization as well

Reply:
Made both changes.